src/blocks/block-anchor.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * A higher-level component that layouts a component
+ * with respect to its parent block.
+ */
+import React, {PropTypes} from 'react';
+
+const {oneOf, any} = PropTypes;
+
+export const HORIZONTAL_ALIGNS = {
+  LEFT: 'left',
+  RIGHT: 'right',
+  CENTER: 'center'
+};
+
+export const VERTICAL_ALIGNS = {
+  TOP: 'top',
+  BOTTOM: 'bottom',
+  MIDDLE: 'middle'
+};
+
+const propTypes = {
+  component: any.isRequired,
+  alignHorizontal: oneOf(Object.values(HORIZONTAL_ALIGNS)),
```

Comment:
You might have issues with object.values. Please make sure that it's transpiled correctly